src/api/sse.ts

```diff
@@ -77,7 +77,7 @@ export interface EventSSEParams extends BaseSSEParams {
 	'name-conjunct'?: boolean;
 }
 
-export interface MessagesSSEParams extends BaseSSEParams {
+export interface MessagesSSEParams extends Omit<BaseSSEParams, 'resumeFromId'> {
```

Comment:
if messages api doesnt accept resumeFromId now maybe we should move this param to EventSSEParams?

Reply:
Sure, your way is better. Fixed: https://github.com/th2-net/th2-rpt-viewer/pull/344/commits/0d9ef297493165d84640d7d127b96199cbcd5717